### ISSUE TITLE
IPFS

### ISSUE THREAD

**Issue by seniorm0ment** [pandabuilder/pandachaika#8]
Suggesting wrapping in IPFS
https://ipfs.io/

**Comment by pandabuilder** [comment]
Yes, I heard about this years ago, but can't really look into it now, due to available server resources and development time.

**Comment by cdzxc** [comment]
I seem to have some files in the “missing archives” list，how can i get in touch with  project staff？

**Comment by Kadantte** [comment]
> I seem to have some files in the “missing archives” list，how can i get in touch with project staff？

Contact 'em Here: `panda at chaika.moe` change `at` to `@`.
